docs/references/javascript/clerk/sign-in.mdx

```diff
@@ -112,111 +119,138 @@ function unmountSignIn(node: HTMLDivElement): void;
 
   const signInComponent = document.querySelector<HTMLDivElement>('#sign-in')!;
 
-  const clerk = new Clerk('pk_[publishable_key]');
-
+  // Initialize Clerk with your Clerk publishable key
+  const clerk = new Clerk(`{{pub_key}}`);

```

Comment:
Why is it double curly braces template literal?

Reply:
because that's how injectkeys works - it injects a value that needs to be a string. doesn't have to be backticks, we can change to quotes